pkg/agent/agent_windows.go

```diff
@@ -250,19 +268,19 @@ func (i *Initializer) prepareOVSBridgeOnHNSNetwork() error {
 		return err
 	}
 	var uplinkPortUUID string
-	uplinkPortUUID, err = i.ovsBridgeClient.CreateUplinkPort(uplink, freePort, nil)
+	uplinkPortUUID, err = i.ovsBridgeClient.CreateUplinkPort(uplinkIFName, freePort, nil)
 	if err != nil {
-		klog.Errorf("Failed to add uplink port %s: %v", uplink, err)
+		klog.Errorf("Failed to add uplink port %s: %v", uplinkIFName, err)
 		return err
 	}
-	uplinkOFPort, err := i.ovsBridgeClient.GetOFPort(uplink, false)
+	uplinkOFPort, err := i.ovsBridgeClient.GetOFPort(uplinkIFName, false)
 	if err != nil {
-		return fmt.Errorf("failed to get uplink ofport %s: err=%w", uplink, err)
+		return fmt.Errorf("failed to get uplink ofport %s: err=%w", uplinkIFName, err)
 	}
-	klog.InfoS("Allocated OpenFlow port for uplink interface", "port", uplink, "ofPort", uplinkOFPort)
+	klog.InfoS("Allocated OpenFlow port for uplink interface", "port", uplinkIFName, "ofPort", uplinkOFPort)
 	i.nodeConfig.UplinkNetConfig.OFPort = uint32(uplinkOFPort)
-	i.nodeConfig.HostInterfaceOFPort = config.BridgeOFPort
-	uplinkInterface := interfacestore.NewUplinkInterface(uplink)
+	i.nodeConfig.HostInterfaceOFPort = config.HostOFPort
```

Comment:
Here `i.nodeConfig.HostInterfaceOFPort` you should use the value read from OVSDB, but not the configured number?

Reply:
Updated. Thanks.